_rules/auto-update-text-efbfc7.md

```diff
@@ -0,0 +1,327 @@
+---
+id: efbfc7
+name: Text content that updates automatically can be paused, stopped or hidden
+rule_type: atomic
+description: |
+  This rule checks that for any auto-updating text content, there are instruments to pause, stop, or hide it or to control its update frequency.
+accessibility_requirements: # Remove whatever is not applicable
+  wcag20:2.2.2: # Pause, Stop, Hide (A)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+  - CSS Styling
+acknowledgments:
+  authors:
+    - Carlos Duarte
+---
+
+## Applicability
+
+The rule applies to any [HTML element][] that has a [visible][] [text node][] as a [descendant][] in the [flat tree][] if:
+
+- **changed:** the `innerText` property of the [element][html element] changes multiple times within a 10 minute time span where there is no [user interaction][]; and
+- **no child changed:** the [element][html element] does not have [children][child] in the [flat tree][] whose `innerText` property also changes; and
+- **not alone:** the [element][html element] has an [ancestor][] element in the [flat tree][] with a non-empty `innerText` property whose value is different from the `innerText` of the test target.
+
+**Note:** The 10 minute time span is an arbitrary limit which is not included in WCAG. Content that updates less frequently, or that doesn't update on a regular interval, will be inapplicable for this rule but may nonetheless fail [Success Criterion 2.2.2: Pause, Stop, Hide][sc 2.2.2]. The accessibility problem tends to be less severe for longer time periods, and without a time limit, testing this rule consistently would be impractical.
+
+## Expectation 1
+
+For each test target there exists a set of [instruments][instrument] to achieve one of the following objectives:
```

Comment:
The word "exists" is still a little problematic here. This isn't something that can be defined. I realise you're limiting where it exists in expectation 2, but that still makes expectation 1 unanswerable. I think you may have to combine the two expectations in some way.

Reply:
Yes, I agree that we probably need to close the set comprehension (i.e. combine with expectation 2) to avoid the following scenario:

1. Here's a target I've found on www.somewhere.org
2. Hey, here's an instrument to stop it, it's on www.elsewhere.com, so Expectation 1 passes.
3. Wait, there is absolutely nothing on www.somewhere.org that tells me to go to www.elsewhere.com, so the instrument is not in a clearly labeled location and Expectation 2 fails

Without combining, Expectation 1 is basically allowed to look anywhere on the web.

Reply:
I've suggested a possible combination of both expectations. Please review.